docs/source/examples/Widget Asynchronous.ipynb

```diff
@@ -282,6 +282,213 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "c5a24b1c6cbe4089bbdc5d36c6edac4f",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/html": [
+       "<p>Failed to display Jupyter Widget of type <code>interactive</code>.</p>\n",
+       "<p>\n",
+       "  If you're reading this message in Jupyter Notebook or JupyterLab, it may mean\n",
+       "  that the widgets JavaScript is still loading. If this message persists, it\n",
+       "  likely means that the widgets JavaScript library is either not installed or\n",
+       "  not enabled. See the <a href=\"https://ipywidgets.readthedocs.io/en/stable/user_install.html\">Jupyter\n",
+       "  Widgets Documentation</a> for setup instructions.\n",
+       "</p>\n",
+       "<p>\n",
+       "  If you're reading this message in another notebook frontend (for example, a static\n",
+       "  rendering on GitHub or <a href=\"https://nbviewer.jupyter.org/\">NBViewer</a>),\n",
+       "  it may mean that your frontend doesn't currently support widgets.\n",
+       "</p>\n"
+      ],
+      "text/plain": [
+       "interactive(children=(Text(value='WHATEVER', description='Whatever'), Output()), _dom_classes=('widget-interact',))"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import datetime\n",
+    "import threading\n",
+    "import time\n",
+    "\n",
+    "from IPython.display import (\n",
+    "    HTML,\n",
+    "    display,\n",
+    ")\n",
+    "import ipywidgets\n",
+    "\n",
+    "def alpha(whatever):\n",
+    "\n",
+    "    def beta(something, container, timerlength=1.5, interval=0.1):\n",
+    "        ctr = 0\n",
+    "        \n",
+    "        pre_output = ipywidgets.Output()\n",
+    "        with pre_output:\n",
+    "            display('Start beta() at {}'.format(time.time()))\n",
+    "        container.children += (pre_output, )\n",
+    "            \n",
+    "        while ctr < timerlength:\n",
+    "            time.sleep(interval)\n",
+    "            ctr += interval\n",
+    "            \n",
+    "        post_output = ipywidgets.Output()\n",
+    "        with post_output:\n",
+    "            display(\"beta() One {} at {}\".format(something, time.time()))\n",
+    "            display(\"beta() Two {} at {}\".format(something, time.time()))\n",
+    "            display(\"beta() Thr {} at {}\".format(something, time.time()))\n",
+    "            display(\"beta() Fou {} at {}\".format(something, time.time()))\n",
+    "            display(\"beta() Fiv {} at {}\".format(something, time.time()))        \n",
```

Comment:
Can we not just use `print` instead of `display` here?

Reply:
print seems to work for me, whereas display gives the issue we've seen.

Reply:
Yeah, that's my experience as well - I wasn't able to trigger the problem with `print()`, but I was able to trigger it with `display()`.